README.md

```diff
@@ -23,10 +23,10 @@ Parsing and verifying tokens is pretty straight forward.  You pass in the token
 ```go
 	token, err := jwt.Parse(myToken, func(token *jwt.Token) (interface{}, error) {
 		// Don't forget to validate the alg is what you expect:
-		if _, ok := token.Method.(*jwt.SigningMethodRSA); !ok {
+		if _, ok := token.Method.(*jwt.SigningMethodHMAC); !ok {
 			return nil, fmt.Errorf("Unexpected signing method: %v", token.Header["alg"])
 		}
-		return myLookupKey(token.Header["kid"]), nil
+		return []byte(mySigningKey), nil
```

Comment:
This might very well be a common use case. I do believe, however, that it is not that clear what "myLookupKey" should be.